proto/cometbft/abci/v2/service.proto

```
@@ -1,9 +1,9 @@
 syntax = "proto3";
-package cometbft.abci.v1;
+package cometbft.abci.v2;
 
-import "cometbft/abci/v1/types.proto";
+import "cometbft/abci/v2/types.proto";
```

Buf (1.54.0)

import "cometbft/abci/v2/types.proto": file does not exist

(COMPILE)